src/MongoDB.Bson/MongoDB.Bson.csproj

```diff
@@ -12,6 +12,10 @@
   <PropertyGroup>
     <DefineConstants>TRACE</DefineConstants>
   </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(TargetFramework)'=='netstandard1.5'">
+    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+  </PropertyGroup>
```

Comment:
We'd probably compile and pass all tests without these lines, but...

We currently depend on version 1.6 of NetStandard.Library so it would be better to not change that until we start targetting new versions of netstandard.